trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -177,9 +177,13 @@ describe(page, () => {
 
     wrapper.expectElement(`${keyingSheetTabSelector} div.govuk-button-group`).toExist();
 
-    wrapper.expectPrimaryButton(`${keyingSheetTabSelector} [data-cy="keying-sheet-mark-as-done-button"]`).toLinkTo('#', 'Mark as done');
+    const markAsDoneButtonSelector = `${keyingSheetTabSelector} [data-cy="keying-sheet-mark-as-done-button"]`;
```

Comment:
Could we also test the classes of these two buttons is as expected (done as primary, to do as secondary)?